client/me/help/help-contact/index.jsx

```diff
@@ -692,6 +692,7 @@ export default connect(
 			ticketSupportEligible: isTicketSupportEligible( state ),
 			ticketSupportRequestError: getTicketSupportRequestError( state ),
 			hasMoreThanOneSite: getCurrentUserSiteCount( state ) > 1,
+			shouldStartHappychatConnection: ! isRequestingSites( state ) && helpSelectedSiteId,
```

Comment:
> Could we consider to make a new selector of this and add a test case for it?
 
Not sure if testing a selector made out of two tested selectors really adds much value.

> What if we specifically check if the site's data is already loaded instead of checking if the sites' data is being requested? Something like:

That's a good point but I think there are things down the stream that rely on other selectors as well (like `getRawSite` in `isJetpackSite` I think) and also `isATEnabled` so instead of checking all of them I would rather check if we have sites and assume the other selectors using the site are tested and work correctly.